src/Pages/LogInSignup.jsx

```diff
@@ -39,34 +44,38 @@ function LoginSignup() {
       setIsLoggedIn(true);
 
       try {
-        const response = await axios.get(`${url}/users`, {
-          params: { email: user.email, password: user.password },
+        const userCredentials = await signInWithEmailAndPassword(
+          auth,
+          user.email,
+          user.password
+        );
+        setAuthUser(userCredentials.user);
+        const response = await axios.post(`${url}users/login`, {
+          email: user.email,
         });
-
-        const foundUser = response.data[0];
-
-        if (foundUser) {
-          setUserData(foundUser);
-        } else {
-          console.error("User not found");
-          setError("Invalid credentials. Please try again.");
-          setIsLoggedIn(false);
-        }
+        navigate(`/user-profile/${response.data.id}`);
+        // console.log(userCredentials);
+        // console.log(response.data);
```

Comment:
this applies to other parts of the diff but generally not worth committing commented out code